apps/eclipse/src/app/global.css

```
@@ -1,30 +1,139 @@
 @import "tailwindcss";
 @import "@prisma/eclipse/styles/globals.css";
-@import "fumadocs-ui/css/shadcn.css";
-@import "fumadocs-ui/css/preset.css";
-@import "@prisma-docs/ui/styles";
 
-#nd-sidebar {
-    @apply border-e;
-    background-color: var(--color-background);
-}
+@theme {
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)